examples/tv-casting-app/linux/main.cpp

```diff
@@ -156,31 +177,26 @@ void PrepareForCommissioning(const Dnssd::DiscoveredNodeData * selectedCommissio
     if (selectedCommissioner != nullptr)
     {
         // Send User Directed commissioning request
-        ReturnOnFailure(Server::GetInstance().SendUserDirectedCommissioningRequest(chip::Transport::PeerAddress::UDP(
-            selectedCommissioner->ipAddress[0], selectedCommissioner->port, selectedCommissioner->interfaceId)));
+        // Wait 1 second to allow our commissionee DNS records to publish (needed on Mac)
+        int32_t expiration = 1;
+        DeviceLayer::SystemLayer().StartTimer(System::Clock::Seconds32(expiration),
```

Comment:
is it ok to ignore any CHIP_ERROR returned here?

Reply:
addressed